LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2021 University of Oxford
+Copyright (c) {{current_year}} {{organisation}}}
```

Comment:
Quite unimportant, but there's a stray brace here.

There's some risk of repos being made public without this getting changed, is there a "making repos public" checklist that this should be added to?

Reply:
I don't think there is, but there should be!